Source/Initialization/SampleGaussianDistribution.H

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2024 Remi Lehe, Revathi Jambunathan
```

Comment:
Could we call this file `SampleGaussianFluxDistribution` instead of `SampleGaussianDistribution`?
This is because the distribution below is consistently called "gaussian-flux" in WarpX, to distinguish it from a regular Gaussian distribution.